test/cli/fixtures/list/list.json

```
@@ -0,0 +1,26 @@
+[
+  {
+    "name": "basic suite > inner suite > some test",
+    "file": "/Users/sheremet.mac/Projects/vitest/test/cli/fixtures/list/basic.test.ts"
```

Let's change these to something generic

```suggestion
    "file": "/fixtures-root/vitest/test/cli/fixtures/list/basic.test.ts"
```